com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Math/Interpolation/SmoothstepNode.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class SmoothstepNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "Smoothstep",
+            "Out = smoothstep(Edge1, Edge2, In);",
+            new ParameterDescriptor("Edge1", TYPE.Vector, Usage.In),
+            new ParameterDescriptor("Edge2", TYPE.Vector, Usage.In, new float[] { 1f, 1f, 1f, 1f }),
+            new ParameterDescriptor("In", TYPE.Vector, Usage.In),
```

Comment:
How would you feel about moving the In port to the top here so that it's consistent with having In always at the top?